acceptance/acceptance_test.go

```diff
@@ -306,15 +306,15 @@ func getSkipReason(config *internal.TestConfig, configPath string) string {
 		}
 
 		if isTruePtr(config.RequiresUnityCatalog) && os.Getenv("TEST_METASTORE_ID") == "" {
-			return fmt.Sprintf("Disabled via RequiresUnityCatalog setting in %s (TEST_METASTORE_ID=%s)", configPath, os.Getenv("TEST_METASTORE_ID"))
+			return fmt.Sprintf("Disabled via RequiresUnityCatalog setting in %s (TEST_METASTORE_ID is empty)", configPath)
```

Comment:
We use this pattern VARIABLE=%s (with %s potentially empty) in other places as well. 

I don't see why change it -- it's a debugging information, the important thing here is seeing the value. Furthermore if someone refactors the condition but not the log message, the log message will still output the value correctly.

Reply:
While we do use the pattern VARIABLE=%s in other places, in this specific case the conditional ensures that VARIABLE will always be empty when that log line is reached. This means the output would consistently show `TEST_METASTORE_ID=` (and other variable names) without providing meaningful debugging information. 

Additionally, any future refactoring that changes the conditional logic would necessarily touch this code block, so the log message could be updated appropriately at that time if needed.